snippets/fer1_annoted.js

```diff
@@ -54,5 +56,51 @@ var defMenu = new msa.menu.defaultmenu({
 });
 defMenu.render();
 
+// pin features
+const pinnedFeatures = [
+  {
+      "feature": "gene",
+      "start": 0,
+      "end": 100,
+      "attributes": {
+          "Name": "VH",
+          "Color": "#E5FCDD",
+          "textColor": "#71B567",
+        }
+      },
+      {
+        "feature": "gene",
+        "start": 0,
+        "end": 20,
+        "attributes": {
+          "Name": "HFR1",
+          "Color": "#E5FCDD",
+          "textColor": "#71B567",
```

Comment:
There's some inconsitency with the casing here.
I believe we want `camelCase` everywhere, but the old code uses `PascalCase`.

We can fix this later with a follow up since it's not too important.